### DISCUSSION TITLE
Label support

### DISCUSSION THREAD

**Discussion by straussmaximilian** [MannLabs/alphapept#322] [Ideas]
AlphaPept could be extended and improved to support various labels and different instruments, e.g.:

- SILAC for Bruker / Thermo
- TMT for Bruker / Thermo

Would you please upvote this idea if you think this is relevant? Also what kind of labelling approach you would find most useful?

**Comment by mgleeming** [comment]
Adding support for TMT-based quant would be great! As for which labelling approach - we run way more TMT than SILAC so TMT would be my vote :)

**Comment by michaelsteidel86** [comment]
Both SILAC and TMT/pro would be much appreciated! And of these already covered, or planned?
